src/components/PlayerInfo/VoteCard/VoteCard.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from '../PlayWordAndCard/PlayWordAndCard.module.css';
+
+export class VoteCard extends React.PureComponent {
```

Comment:
This component doesn't have any state. If you want to optimise, you should make this a function component, rather than a Pure component.